apps/website/package.json

```diff
@@ -35,6 +35,7 @@
     "@next-auth/prisma-adapter": "^1.0.7",
     "@next/env": "^13.5.1",
     "@paralleldrive/cuid2": "^2.2.2",
+    "@planetscale/database": "^1.11.0",
```

Comment:
Note: Using this database access through http on edge functions will only work with planetscale and not with other mysql hosters. So we probably should look for another solution instead. Just used it 'cause its easy and has very low runtime cost.